src/commonMain/kotlin/fr/acinq/lightning/blockchain/electrum/ElectrumClientExtensions.kt

```diff
@@ -1,16 +1,6 @@
 package fr.acinq.lightning.blockchain.electrum
 
-import fr.acinq.bitcoin.Satoshi
 import fr.acinq.bitcoin.Transaction
-import fr.acinq.bitcoin.TxId
-import fr.acinq.lightning.blockchain.fee.FeeratePerKw
-import fr.acinq.lightning.channel.Commitments
-import fr.acinq.lightning.channel.LocalFundingStatus
-import fr.acinq.lightning.logging.*
-import fr.acinq.lightning.transactions.Transactions
-import fr.acinq.lightning.utils.sat
-
-suspend fun IElectrumClient.getConfirmations(txId: TxId): Int? = getTx(txId)?.let { tx -> getConfirmations(tx) }
```

Comment:
Do we really need this file? It's a bit annoying to navigate implementations, this should directly be in the `ElectrumClient`?

Reply:
We don't, removed in c4d0218e.